app/decorators/models/decidim/meeting_decorator.rb

```diff
@@ -0,0 +1,13 @@
+# frozen_string_literal: true
+
+Decidim::Meetings::Meeting.class_eval do
+  scope :thirteen_am, -> { where("start_time::time < ?", "13:00:00") }
```

Comment:
We must be aware of the TimeZone and the daylight savings. Better create a Ruby Time object at 13:00 and pass it as parameter.

Reply:
Can you check if it is correct now?